### ISSUE TITLE
Support APAC server

### ISSUE THREAD

**Issue by audreylim** [gagebenne/pydexcom#71]
`pydexcom` currently supports 2 Dexcom Share endpoints: US and non-US servers. The non-US server is the EU server and so users on the APAC server in Japan cannot authenticate.

Suggestion:

Add
```
DEXCOM_APPLICATION_ID_APAC: str = "d8665ade-9673-4e27-9ff6-92db4ce13d13"

DEXCOM_BASE_URL_APAC: str = "https://share.dexcom.jp"
"""Dexcom Share API base url for APAC."""

 def __init__(
        self,
        *,
        password: str,
        account_id: str | None = None,
        username: str | None = None,
        ous: bool = False,
        apac: bool = False, # Added
    ) -> None: 
```

```
>>> dexcom = Dexcom(username="", password="", ous=True)
>>> dexcom = Dexcom(username="", password="", apac=True)
```

To find account id on JP server, go to `uam.dexcom.jp`.

Let me know if you need help with a PR.

**Comment by audreylim** [comment]
Tests with JP account:

```
➜  ~ curl -X POST https://share.dexcom.jp/ShareWebServices/Services/General/AuthenticatePublisherAccount -H "Content-Type: application/json" -d '{"applicationId": "d8665ade-9673-4e27-9ff6-92db4ce13d13", "accountName":"xx@gmail.com", "password":"xx"}'
"<UUID>"

➜  ~ curl -X POST https://shareous1.dexcom.com/ShareWebServices/Services/General/AuthenticatePublisherAccount -H "Content-Type: application/json" -d '{"applicationId": "d89443d2-327c-4a6f-89e5-496bbb0317db", "accountName":"xx@gmail.com", "password":"xx"}'
{"Code":"AccountPasswordInvalid","Message":"Publisher account password failed","SubCode".....
```

**Comment by gagebenne** [comment]
This is awesome; thanks for the thorough suggestion. I'll get started on this tonight, I don't imagine a PR being too far off.

**Comment by gagebenne** [comment]
Hi @audreylim -- could you please try out the alpha release and let me know if everything is functional in APAC?
```bash
pip install pydexcom==0.4.1a0
```

**Comment by audreylim** [comment]
Works great, thanks!

**Comment by peejaychilds** [comment]
There may be some confusion for users calling it 'apac' as Australian users are hosted on the EU endpoint.

Awesome contribution and great project 10/10

**Comment by gagebenne** [comment]
Ah, good to know. I think I will alter the region to just `"jp"` as opposed to `"apac"`. Thanks @peejaychilds!

**Comment by gagebenne** [comment]
Done, thanks y'all! Just published [0.4.1](https://pypi.org/project/pydexcom/0.4.1/)!
